applications/dashboard/controllers/Api/TokensApiController.php

```diff
@@ -0,0 +1,316 @@
+<?php
+/**
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license GPLv2
+ */
+
+use Garden\Schema\Schema;
+use Garden\Web\Exception\ClientException;
+use Garden\Web\Exception\NotFoundException;
+
+/**
+ * API Controller for the `/tokens` resource.
+ */
+class TokensApiController extends AbstractApiController {
+
+    /** Default expiry for issued tokens. */
+    const DEFAULT_EXPIRY = '10 years';
```

Comment:
From a security perspective, I'm concerned about tokens virtually not expiring on their own.

What will they be used for?